src/provider/types/abci.ts

```diff
@@ -47,4 +47,12 @@ export interface ABCIAccount {
   };
 }
 
+export interface ABCIResponseSimulateTx {
```

Comment:
I think we have `ResponseBase` somewhere with these values?

Reply:
I found `DeliverTx`. https://github.com/gnolang/tm2-js-client/pull/192/commits/9efdbe70b3b575d681d8a1d8cd58d0473c525363